src/cider-velox/benchmark/CMakeLists.txt

```diff
@@ -106,16 +106,17 @@ set(CIDER_BENCHMARKS_SUPPLEMENT_DEPENDENCIES
     cider_plan_transformer
     cider_plan_parser)
 
-add_executable(tpch_benchmark_velox_plugin TpchBenchmarkCompare.cpp)
+# add_executable(tpch_benchmark_velox_plugin TpchBenchmarkCompare.cpp)
 
-target_link_libraries(
-  tpch_benchmark_velox_plugin ${VELOX_BENCHMARKS_DEPENDENCIES}
-  ${CIDER_BENCHMARKS_SUPPLEMENT_DEPENDENCIES})
+# target_link_libraries( tpch_benchmark_velox_plugin
+# ${VELOX_BENCHMARKS_DEPENDENCIES} ${CIDER_BENCHMARKS_SUPPLEMENT_DEPENDENCIES})
 
-add_executable(CiderOperatorBenchmark CiderOperatorBenchmark.cpp)
-target_link_libraries(CiderOperatorBenchmark ${VELOX_BENCHMARKS_DEPENDENCIES}
-                      ${CIDER_BENCHMARKS_SUPPLEMENT_DEPENDENCIES})
+# add_executable(CiderOperatorBenchmark CiderOperatorBenchmark.cpp)
+# target_link_libraries(CiderOperatorBenchmark ${VELOX_BENCHMARKS_DEPENDENCIES}
+# ${CIDER_BENCHMARKS_SUPPLEMENT_DEPENDENCIES})
```

Comment:
Why disable these benchmarks?

Reply:
because these benchmarks are not fixed for new API，they can't be compiled now